wherescape/ws_env_template.py

```diff
@@ -0,0 +1,134 @@
+            os.environ["WSL_SRC_DBID"] = ""
+            os.environ["WSL_SRC_DB"] = "__db_name__"
+            os.environ["WSL_SRC_SCHEMA"] = "__schema_name__"
+            os.environ["WSL_SRC_USER"] = "__user_name__"
+            os.environ["WSL_SRC_PWD"] = "__pasword__"
+        else:
+            os.environ["WSL_SRC_DSN"] = "wswh dev1"
+            os.environ["WSL_SRC_DSN_ARCH"] = "64"
+            os.environ[
+                "WSL_SRC_SERVER"
+            ] = "xxxxxx.xxxxxx.eu-central-1.rds.amazonaws.com"
+            os.environ["WSL_SRC_DBPORT"] = ""
+            os.environ["WSL_SRC_DBID"] = ""
+            os.environ["WSL_SRC_DB"] = "__db_name__"
+            os.environ["WSL_SRC_SCHEMA"] = ""
+            os.environ["WSL_SRC_USER"] = "__user_name__"
+            os.environ["WSL_SRC_PWD"] = "__pasword__"
+    else:
+        # for directly called hosts scripts
+        os.environ["WSL_LOAD_FULLNAME"] = ""
+        os.environ["WSL_LOAD_TABLE"] = ""
+        os.environ["WSL_LOAD_SCHEMA"] = ""
+        os.environ["WSL_LOAD_DB"] = ""
+
+        os.environ["WSL_TGT_DSN"] = ""
+        os.environ["WSL_TGT_DSN_ARCH"] = "32"
+        os.environ["WSL_TGT_SERVER"] = ""
+        os.environ["WSL_TGT_DBPORT"] = ""
+        os.environ["WSL_TGT_DBID"] = ""
+        # These were missing in .bat file
+        #    os.environ['WSL_TGT_DB']="warehouse"
+        #    os.environ['WSL_TGT_SCHEMA']=""
+        os.environ["WSL_TGT_USER"] = ""
+        os.environ["WSL_TGT_PWD"] = ""
+
+        os.environ["WSL_SRC_DSN"] = ""
+        os.environ["WSL_SRC_DSN_ARCH"] = "32"
+        os.environ["WSL_SRC_SERVER"] = ""
+        os.environ["WSL_SRC_DBPORT"] = ""
+        os.environ["WSL_SRC_DBID"] = ""
+        os.environ["WSL_SRC_DB"] = ""
+        os.environ["WSL_SRC_SCHEMA"] = ""
+        os.environ["WSL_SRC_USER"] = ""
+        os.environ["WSL_SRC_PWD"] = ""
+
+    os.environ["WSL_BINDIR"] = "C:\\Program Files\\WhereScape\\RED\\"
+    # os.environ['WSL_WORKDIR'] = f"C:\\WhereScape\\Scheduler\\Work\\{capped_environment}\\"
+    os.environ[
+        "WSL_WORKDIR"
+    ] = "C:\\Users\\Bart\\Documents\\GitHub\\wherescape-warehouse\\"
+
+    os.environ["WSL_SEQUENCE"] = "1234"
+    os.environ["WSL_JOB_KEY"] = "9999"
+    os.environ["WSL_JOB_NAME"] = "Local env"
+    os.environ["WSL_TASK_KEY"] = "6666"
+    os.environ["WSL_TASK_NAME"] = tablename
+
+    os.environ["WSL_TEMP_DB"] = ""
+
+    # setup passwords for warehouse.py
+    git_dir = r"C:\WhereScape\GitlabRepository"
+
+    os.environ["CONFIG_JSON_PATH"] = (
+        git_dir + r"\wherescape-warehouse\python\config.wherescape.json"
+    )
+
+    # -- set passwords
+    # -- script can run in all environments
+    os.environ["WSWH_L_ETL"] = "__pasword__"
+    os.environ["WSWH_D1_ETL"] = "__pasword__"
+    os.environ["WSWH_D2_ETL"] = "__pasword__"
+    os.environ["WSWH_A_ETL"] = "__pasword__"
+    os.environ["WSWH_P_ETL"] = "__pasword__"
+
+
+if __name__ == "__main__":
+    setup_env("load_hsv_feedback_submissions")
```

Comment:
This env name seems wrong

Reply:
Fixed. Clarified calling the function with more generic parameters.